ci/package.json

```diff
@@ -5,9 +5,8 @@
   "devDependencies": {},
   "dependencies": {
     "browser-sync": "^2.26.3",
-    "cypress": "^4.2.0",
-    "cypress-wait-until": "^1.7.1",
-    "cypress-file-upload": "^3.5.3"
+    "cypress": "^9.1.1",
```

Comment:
Ik vermoed eigenlijk dat er met deze `package.json` (op Jenkins?) niets gedaan wordt, en dat je eerder hier https://github.com/kanselarij-vlaanderen/kaleidos-cypress/blob/30f99c62f511c1d4f4917c6bd14944b07bf710e4/Dockerfile#L1 moet zijn. Zou dat kunnen?

Reply:
Zie ook https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/Jenkinsfile.ci#L83

Reply:
moet eerlijk gezegd toegeven dat ik niet wist dat die kaleidos-cypress repo bestond...

Reply:
Enig idee of (en door wie/wat) de `ci/package.json` gebruikt wordt?